src/vmdtab.h

```diff
@@ -107,6 +110,27 @@ class VMdTab : public VEditTab
 
     bool expandRestorePreviewArea();
 
+    // github image hosting
+    // GitHub identity authentication
+    void githubImageBedAuthentication(QString token);
```

Comment:
I wonder if we could use two separate class to maintain the logic of GitHub and WeChat services. It would make `VMdTab` more clean and small, and make the code more maintainable.

Reply:
Ok, I have extracted the code of the image hosting separately into a class.

Reply:
Should I initiate a new PR?